hosting/technical/data_request.md

```diff
@@ -29,12 +29,13 @@ levels = sorted({1,5,20,150,250,750}.union(tr,lt,ua))
 
 HEALPix grids have `12*4**level` cells, so a level 9 HEALPix grid consists of roughly 3 million cells. It has proven very beneficial for the analysis to also store all lower grid resolutions. This adds approximately 30% to the output volume, and allows prototyping analyses at lower resolution, or generating maps from an amount of data that actually matches the pixels in a plot.
 For level 9 and all lower levels together, about 4 million floats are needed per 2D slice.
+Furthermore, we want the 2D fields on 6-hourly interval as well, and all fields also daily.
 The totals for storing this data (assuming 4 bytes/float, and 50% compression) are
 
 ```
-3D: 3.3TB
-2D: 2.0TB
-total: 5.3TB
+3D: 4.2TB
+2D: 2.7TB
+total: 6.8TB
 ```
```

Comment:
It may be nice to give the reference number for a 2D field. This way, it would be easier to estimate the storage requirements for other simulations (more levels, higher zoom, ...)